op-proposer/proposer/abi_test.go

```diff
@@ -34,10 +34,7 @@ func setupL2OutputOracle() (common.Address, *bind.TransactOpts, *backends.Simula
 		big.NewInt(10),
 		big.NewInt(2),
 		big.NewInt(0),
-		big.NewInt(0),
-		from,
-		common.Address{0xdd},
-		big.NewInt(100))
+	)
```

Comment:
I don't think this is correct. The constructor of the `L2OutputOracle` changed to accept args like so:
```diff
        uint256 _submissionInterval,
        uint256 _l2BlockTime,
-       uint256 _startingBlockNumber,
-       uint256 _startingTimestamp,
-       address _proposer,
-       address _challenger,
        uint256 _finalizationPeriodSeconds
```

So, I believe this should be:
```suggestion
		big.NewInt(10),
		big.NewInt(2),
		big.NewInt(100))
	)
```

Reply:
Great catch here